apps/backend/src/services/codemodService.ts

```diff
@@ -152,6 +153,12 @@ export class CodemodService {
 
     const data: LongCodemodInfo[] = await Promise.all(
       codemods.map(async (codemod) => {
+        let frameworkVersion = null;
+        const match = codemod.name.match(/(\d+(?:\.\d+)*)/);
+        if (match) {
+          frameworkVersion = match[1];
+        }
+
```

Comment:
please move all that code to separate function and use it on both places
also you can simplify with optional chaining
codemod.name.match(/(\d+(?:\.\d+)*)/)?.[0]